labelbox/data/annotation_types/geometry/line.py

```diff
@@ -1,10 +1,12 @@
-from typing import List, Optional, Union, Tuple
+from typing import List, Optional, Union, Tuple, TYPE_CHECKING
 
 import geojson
 import numpy as np
 import cv2
```

Comment:
Open cv is an extra dep. You can see them all listed here.
https://github.com/Labelbox/labelbox-python/blob/2c688a7d69bce36984d7237b2d85e8af05aa8e29/setup.py#L27-L30

I would recommend pip installing from your branch from collab without the data package and try importing from annotation_import to see if an error is thrown to make sure this all works.

Reply:
Tested that this works on both colab without the `labelbox[data]` as well as in a minimal docker container that does not have `data` dependencies installed.